website/data.yml

```diff
@@ -321,6 +321,18 @@ categories:
       repo_url: https://github.com/openai/codex
       tags:
       - coding
+    - name: crewAI
+      description: Framework for orchestrating role-playing, autonomous AI agents.
+        By fostering collaborative intelligence, CrewAI empowers agents to work together
+        seamlessly, tackling complex tasks.
+      homepage_url: https://www.crewai.com/
+      logo: crewai.svg
+      repo_url: https://github.com/crewAIInc/crewAI
+    - name: fast-agent
+      description: Define, Prompt and Test MCP enabled Agents and Workflows
+      homepage_url: https://github.com/evalstate/fast-agent
+      logo: fast-agent.png
```

Comment:
can we use the default.png, the logo seems not like an official one. cc @samzong

Reply:
It's really of this. 😂

use `default.png` is ok.